.github/workflows/deploy-pages.yml

```
@@ -0,0 +1,49 @@
+        with:
+          path: docs/dist
+
+  deploy:
+    needs: build
+    runs-on: ubuntu-latest
+    permissions:
+      pages: write
+      id-token: write
+    steps:
+      - name: Deploy to GitHub Pages
+        uses: actions/deploy-pages@v1
```

actionlint (1.7.11)

[error] the runner of "actions/deploy-pages@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)